nscplugin/src/main/scala-3/scala/scalanative/nscplugin/NirGenExpr.scala

```diff
@@ -2091,6 +2104,33 @@ trait NirGenExpr(using Context) {
       buf.stackalloc(nir.Type.Byte, unboxed, unwind)(using app.span)
     }
 
+    def genSafeZoneAlloc(app: Apply): Val = {
+      val Apply(_, List(tree, sz)) = app
```

Comment:
`sz` is the first parameter and `tree` is the second. The `SafeZoneTest` unit test should pass after fixing this :)

```suggestion
      val Apply(_, List(sz, tree)) = app
```

Reply:
Oh, that's one of the relicts of replacing 
`def withsafeZone(zone: Zone, obj: T)`
with 
`def withSafeZone(obj: T)(using zone: Zone)`
I though I reverted that
That's probably a reason for the failures of CI.